benchmark/write.go

```diff
@@ -27,26 +27,36 @@ func main() {
 	defer f.Close()
 
 	for i := 1; i <= 100; i++ {
-		dataset := String(5)
-		f.WriteString("insert into DATASETS values (" + strconv.Itoa(i) + ", " + "\"" + dataset + "\"" + ");\n")
+		datasetID := i
+		dataset := "/" + String(3) + "/" + String(3) + "/" + String(3)
+		putDataset := fmt.Sprintf("insert into DATASETS values (%d, %s); \n", datasetID, "\""+dataset+"\"")
```

Comment:
why not to put quotes inside the statements? E.g.
```
fmt.Sprintf("insert into DATASETS values (%d, \"%s\"); \n", datasetID, dataset)
```
is easier to read (especially when you pass multiple parameters, like in block/files insert statements.
Please adjust all of them accordingly.